### PR TITLE
(Breaking change) Rename dash to console

### PR DESCRIPTION
**Pull request by chainlink**

### CODE REVIEW

**Review by armaanv — CHANGES_REQUESTED** [review]
I think the TestconsoleImageAndConfigTag test will fail due to the wrong expected image tag. Looks like CI didn't run that test.

LGTM otherwise

File: etc/helm/test/pachyderm_helm_test.go [hunk 1]
```diff
@@ -19,29 +19,29 @@ import (
...
 	}
 
-	output := helm.RenderTemplate(t, options, helmChartPath, "deployment", []string{"templates/dash/deployment.yaml"})
+	output := helm.RenderTemplate(t, options, helmChartPath, "deployment", []string{"templates/console/deployment.yaml"})
 
 	var deployment appsv1.Deployment
 	helm.UnmarshalK8SYaml(t, output, &deployment)
 
-	expectedContainerImage := "pachyderm/haberdashery:abc123"
+	expectedContainerImage := "pachyderm/haberconsoleery:abc123"
```
[the remark this answers is not in the document]
Comment (chainlink) [reply etc/helm/test/pachyderm_helm_test.go:44]: lol haberconsoleery

File: etc/helm/test/tls_test.go [hunk 2]
```diff
@@ -72,30 +72,30 @@ func TestEnablePachTLSExistingSecret(t *testing.T) {
...
 			"deployTarget":        "LOCAL",
-			"dash.enabled":        "true",
+			"console.enabled":     "true",
 			"ingress.tls.enabled": "true",
 			"ingress.host":        "http://blah.com",
 		},
 	}
 
-	_, err := helm.RenderTemplateE(t, options, helmChartPath, "secret", []string{"templates/dash/ingress.yaml"})
+	_, err := helm.RenderTemplateE(t, options, helmChartPath, "secret", []string{"templates/ingress/ingress.yaml"})
```
Comment (armaanv) [inline etc/helm/test/tls_test.go:86]: should this be console?

Comment (chainlink) [reply etc/helm/test/tls_test.go:86]: ingress has been moved to root

**Review by armaanv — APPROVED** [review]

**Pull request merged**